rotkehlchen/chain/evm/decoding/decoder.py

```diff
@@ -93,6 +94,41 @@
 MIN_LOGS_PROCESSED_TO_SLEEP = 1000
 
 
+def decoding_wrapper(
```

Comment:
Perhaprs a better name?

```suggestion
def decode_safely(
```